orientdb/src/test/java/org/hibernate/ogm/datastore/orientdb/test/jpa/OrientDbQueryParamTest.java

```diff
@@ -43,11 +38,10 @@
  */
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @RunWith(Parameterized.class)
-public class OrientDbQueryParamTest {
+public class OrientDbQueryParamTest extends OgmJpaTestCase {
```

Comment:
This test isn't working for me, I have a NullPointerException with the method getFactory

Also, why do you need to fix the method order?